pkg/runtime/runtime.go

```diff
@@ -489,12 +491,20 @@ func (a *DaprRuntime) sendBatchOutputBindingsSequential(to []string, data []byte
 	return nil
 }
 
-func (a *DaprRuntime) sendToOutputBinding(name string, req *bindings.WriteRequest) error {
+func (a *DaprRuntime) sendToOutputBinding(name string, req *bindings.InvokeRequest) (*bindings.InvokeResponse, error) {
+	if req.Operation == "" {
+		return nil, errors.New("operation field is missing from request")
+	}
+
 	if binding, ok := a.outputBindings[name]; ok {
-		err := binding.Write(req)
-		return err
+		for _, o := range binding.Operations() {
+			if o == req.Operation {
+				return binding.Invoke(req)
+			}
+		}
+		return nil, fmt.Errorf("binding %s does not support operation %s", name, req.Operation)
```

Comment:
In addition to this can it return supported operations as well, something like:
binding %s does not support operation %s, supported operations for binding %s are %s"

Reply:
Done